### ISSUE TITLE
Tests missing from 0.5.5 source tarball

### ISSUE THREAD

**Issue by danc86** [matiasb/python-unidiff#51]
The following files are missing from the 0.5.5 tarball (compared with previous tarballs):

```
tests/__init__.py
tests/test_hunks.py
tests/test_parser.py
tests/test_patchedfile.py
```

The test sample files are still there, from #23, it's just the test *code* which is missing this time around.

I suspect it could be due to the removal of `test_suite='tests'` from setup.py in 0.5.5, but honestly setuptools just makes me confused and angry every time I look at it so I'm not sure if that's right.

**Comment by matiasb** [comment]
I see :-/
I'll fix and prepare a new release, as you said adding `test_suite='tests'` to the setup call does the trick (didn't notice I missed this with the setup.py refactoring).

**Comment by matiasb** [comment]
It should be fixed in 71f510b181dbb3bb3885fe032e81b0d3258b4287. Thanks!
I would like to work on a few other changes so will hold a new release for now.
